src/main/src/main/java/org/geoserver/security/decorators/DecoratingWMSStoreInfo.java

```diff
@@ -156,6 +156,26 @@ public void setPassword(String password) {
         delegate.setPassword(password);
     }
 
+    @Override
+    public String getHeaderName() {
+        return delegate.getHeaderName();
+    }
+
+    @Override
+    public void setHeaderName(String headerName) {
+        delegate.setHeaderName(headerName);
+    }
+
+    @Override
+    public String getHeaderValue() {
+        return delegate.getHeaderValue();
+    }
+
+    @Override
+    public void setHeaderValue(String headerValue) {
+        delegate.setHeaderName(headerValue);
```

Comment:
This should be delegate.setHeader**Value**(headerValue) - fixed in the next commit